main.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from PyQt5 import QtWidgets
+import InitWindow
+
+class InitWindow(QtWidgets.QMainWindow, InitWindow.Ui_InitWindow):
```

Comment:
Преобразуй класс в виджет (Наследуйся от QWidget), чтоб его можно было свопать из главного окна с помощью различных событий будь то нажатия твоих кнопок, кнопок менюбара или смена состояния игры.

```suggestion
class InitWindow(QtWidgets.QWidget, InitWindow.Ui_InitWindow):
```